services/enrichment/manager/src/lib/checks.js

```diff
@@ -5,16 +5,14 @@ export const canLocationsBeEnriched = locations =>
       return false;
     }
 
-    // Centroid is empty but country code is provided
-    const needsLocation = !loc.centroid && loc.country_code;
-    if (needsLocation) return true;
-
-    // Country code is empty but centroid is provided
-    const needsCountryCode = !loc.country_code && loc.centroid;
-
-    if (needsCountryCode) return true;
-
-    return false;
+    // Centroid is empty but country code is provided OR
+    // Country code is empty but centroid is provided OR
+    // there are no NUTS code provided
```

Comment:
`there are no NUTS code` is not enough to tell if the record can be enriched or not